cmd/config/metrics-report.yml

```diff
@@ -224,6 +224,788 @@
   metricName: max-memory-stackrox
   instant: true
 
+# ACS - Central
```

Comment:
I don't think we should add ACS metrics in this file, metrics here are for standard OCP deployments, how are you using it? There's not any ACS related workload in this repository

Reply:
If you're using this metrics file from an external repo I recommend you to create a new metrics file with all the acs/stackrox stuff rather than modifying this one